src/test/skript/tests/syntaxes/effects/EffOperations.sk

```diff
@@ -0,0 +1,68 @@
+test "operations numbers":
```

Comment:
needs tests where both values are variables, where the other value is not a valid option, where the value is a variable with an invalid type, etc. Edge cases need to be covered.